src/tribol/interface/tribol.cpp

```diff
@@ -921,6 +1110,96 @@ integer update( integer cycle, real t, real &dt )
       }
 
       CouplingScheme* couplingScheme  = csManager.getCoupling(csIndex);
+
+      // update redecomp meshes if supplied mfem data
```

Comment:
If this is the first call to update() we create the redecomp meshes right? If not, then we update them fresh? Maybe clarify?

Reply:
This is explicitly done through `updateMfemParallelDecomposition()` now.